test/test_norm_sum_record.py

```diff
@@ -15,6 +15,9 @@
 """Test cases for the NormalisedSummaryRecord based on SummaryRecord tests."""
 
 
+from builtins import zip
+from builtins import str
```

Comment:
Can we collapse these onto one line? (Like the `datetime` imports below.)

If that doesn't cause issues, can you do it in the other places that have more than one `builtins` import?

Reply:
Resolved now. Done